src/domain/cohort.js

```diff
@@ -35,12 +35,45 @@ export class Cohort {
   }
 }
 
-export const getCohortById = (cohortId) => {
-  return dbClient.cohort.findFirst({
+export const getCohortById = async (cohortId) => {
+  const foundCohort = await dbClient.cohort.findUnique({
     where: {
       id: cohortId
+    },
+    include: {
+      users: {
+        select: {
+          id: true,
+          cohortId: true,
+          role: true,
+          email: true,
+          profile: {
+            select: {
+              firstName: true,
+              lastName: true,
+              bio: true,
+              githubUsername: true
+            }
+          }
+        }
+      }
     }
   })
+
+  if (foundCohort) {
+    return { ...foundCohort, users: flatten(foundCohort) }
+  }
+
+  return foundCohort
```

Comment:
Since this is only ever going to be null (or undefined, I can't remember), I think it's more helpful to explicitly `return null` here